lib/src/countly_flutter.dart

```diff
@@ -257,6 +257,11 @@ class Countly {
       }
     }
     if (config.enableUnhandledCrashReporting != null) {
+      FlutterError.onError = _recordFlutterError;
```

Comment:
please add comments here describing what they do for future developers

Reply:
okay